parcels/examples/documentation_MPI.ipynb

```diff
@@ -59,14 +59,14 @@
     "\n",
     "However, in tests we have experienced that this may not necessarily be the most efficient chunking. Therefore, Parcels provides control over the chunk-size via the `field_chunksize` keyword in `Field` creation, which requires a tuple that sets the typical size of chunks for each dimension.\n",
     "\n",
-    "It is strongly encouraged to explore what the best value for chunksize is for your experiment, which will depend  on the `FieldSet`, the `ParticleSet` and the type of simulation (2D versus 3D). As a guidance, we have found that chunksizes in the zonal and meridional direction of approximately `500` are typically most effective."
+    "It is strongly encouraged to explore what the best value for chunksize is for your experiment, which will depend  on the `FieldSet`, the `ParticleSet` and the type of simulation (2D versus 3D). As a guidance, we have found that chunksizes in the zonal and meridional direction of approximately around *128-512* are typically most effective. The binning relates to the size of the model and its data size, so power-of-two values are advantageous but not required."
```

Comment:
change *128-512* to *128 to 512* for better readability?

Reply:
yes, I can do that.

Reply:
still need to do that before merge

Reply:
I've just updated it